extensions/sql-migration/src/constants/strings.ts

```diff
@@ -13,6 +13,12 @@ export const WIZARD_TITLE = localize('sql-migration.wizard.title', "SQL Migratio
 export const SOURCE_CONFIGURATION_PAGE_TITLE = localize('sql.migration.wizard.source_configuration.title', "SQL Source Configuration");
 // //#endregion
 
+// Assessments Progress Page
+export const ASSESSMENT_PROGRESS = localize('sql.migration.assessments.progress', "Assessments Progress");
+export const ASSESSMENT_IN_PROGRESS = localize('sql.migration.assessment.in.progress', "Assessment in progress");
+export function ASSESSMENT_IN_PROGRESS_CONTENT(dbName: string) {
+	return localize('sql.migration.assessment.in.progress.content', "We are assessing the selected databases in your SQL server instance {0} to identify the right Azure SQL target.\n\nThis may take some time.", dbName);
```

Comment:
This won't make any sense at this point in time as we don't have database selector in the previous page.